docs/events-interception.md

```diff
@@ -0,0 +1,190 @@
+# About events interception {#events-interception}
```

Comment:
We have one more page about testplane events here: https://github.com/gemini-testing/testplane/blob/2ced168569b3262137a094bed331b763805a1268/docs/events.md

I think these should either be merged or original `events.md` should include links to the new articles